src/Raven.Server/Documents/Handlers/Debugging/ServerWideDebugInfoPackageHandler.cs

```diff
@@ -511,5 +514,547 @@ public enum DebugInfoPackageContentType
+                            blittable = await context.ReadForMemoryAsync(entry.Open(), "");
+                            if (blittable.TryGet("Results", out BlittableJsonReaderArray clusterTransactionOperations))
+                                clusterTransactionOperationsCount = clusterTransactionOperations.Length;
+
+                            break;
+
+                        case "documents.huge.json":
+                            hugeDocuments = GetHugeDocuments(entry, context);
+                            break;
+
+                        case "admin.memory.gc.json":
+                            blittable = await context.ReadForMemoryAsync(entry.Open(), "");
+                            if (blittable.TryGet("Any", out BlittableJsonReaderObject gcInfo))
+                            {
+                                if (gcInfo.TryGet("PauseDurations", out BlittableJsonReaderArray time))
+                                    gcTime = time[0].ToString();
+                            }
+
+                            break;
+
+                        case "admin.info.tcp.active-connections.json":
+                            blittable = await context.ReadForMemoryAsync(entry.Open(), "");
+                            if (blittable.TryGet("TotalConnections", out long connections))
+                                activeConnections = connections;
+
+                            break;
+
+                        case "database-record.json":
+                            blittable = await context.ReadForMemoryAsync(entry.Open(), "");
+                            if (blittable.TryGet("Topology", out BlittableJsonReaderObject topology) && blittable.TryGet("DatabaseName", out string dbName))
+                            {
+                                if (topology.TryGet("Members", out BlittableJsonReaderArray members))
+                                {
+                                    if (nodeTag == members[0].ToString())
+                                        firstNodeDatabases.Add(dbName);
+                                }
+                            }
+
+                            break;
+
+                        case "admin.node.ping.json":
+                            longPings = GetLongPings(entry, context, electionTimeout, urlsToNodes, nodeTag);
+                            break;
+                    }
+                }
+
+                json["Databases"] = databases;
+                json["Databases Count"] = databases.Count;
+                json["Node Version"] = nodeVersion;
+                json["First Node Databases"] = firstNodeDatabases;
+                json["GC Time"] = gcTime;
+                json["Documents Count"] = documentsCount;
+                json["Tombstones Count"] = tombstonesCount;
+                json["Indexes Count"] = indexesCount;
+                json["Revisions Count"] = revisionsCount;
+                json["Threads Count"] = threadsCount;
+                json["Tasks Count"] = tasksCount;
```

Comment:
I think it is better to have a long name instead of an unclear name.
You should explicitly say this is only for `ResponsibleNode` of the current node